composer/src/main/kotlin/com/gojuno/composer/Main.kt

```diff
@@ -90,7 +90,7 @@ fun main(rawArgs: Array<String>) {
                                         .flatMap { adbDeviceTestRun ->
                                             writeJunit4Report(
                                                     suite = adbDeviceTestRun.toSuite(args.testPackage),
-                                                    outputFile = File(File(args.outputDirectory, "junit4-reports"), "${device.id}.xml")
+                                                    outputFile = File(File(args.outputDirectory, "junit4-reports"), "${device.sanitizedId()}.xml")
```

Comment:
hm, this might break someones report parsing, we should emphasize this in release notes

Reply:
There is also an edge case when filenames for multiple devices will be equal after sanitization e.g. both `123:456` and `123-456`.